framework/core/js/src/@types/translator-icu-rich.d.ts

```diff
@@ -1,4 +1,6 @@
 declare module '@askvortsov/rich-icu-message-formatter' {
+  import Mithril from 'mithril';
```

Comment:
Do type imports work here? I guess because it's a declaration file, it doesn't make any difference anyway.